src/app/services/actions/request-history-action-creators.ts

```diff
@@ -1,20 +1,31 @@
 import { Dispatch } from 'redux';
+
 import { IHistoryItem } from '../../../types/history';
-import { ADD_HISTORY_ITEM_SUCCESS, QUERY_GRAPH_SUCCESS,
-  REMOVE_HISTORY_ITEM_SUCCESS, VIEW_HISTORY_ITEM_SUCCESS } from '../redux-constants';
+import { removeHistoryData } from '../../views/sidebar/history/history-utils';
+import {
+  ADD_HISTORY_ITEM_SUCCESS, QUERY_GRAPH_SUCCESS,
+  REMOVE_HISTORY_ITEM_SUCCESS, VIEW_HISTORY_ITEM_SUCCESS
+} from '../redux-constants';
 
 export function addHistoryItem(response: IHistoryItem): any {
```

Comment:
The `response` variable is mostly used to hold data from a network request. What do you think of renaming this to `historyItem`